core/templates/dev/head/css/oppia.css

```diff
@@ -1064,6 +1064,7 @@ textarea {
 }
 
 .oppia-sidebar-menu-open .oppia-sidebar-menu {
+  height: 100vh;
```

Comment:
I'm actually still getting a slight cut-off on my phone at the very bottom on my phone... I think it may be perhaps due to the header or something but I'm not sure. Try 110vh?

Reply:
Also, alphabetize the attributes

Reply:
@tjiang11 Done!